lib/libutee/include/pta_apdu.h

```diff
@@ -0,0 +1,46 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (C) 2021, Foundries Limited
+ * Author: Jorge Ramirez <jorge@foundries.io>
+ */
+
+#ifndef __PTA_APDU_H
+#define __PTA_APDU_H
+
+#define PTA_APDU_UUID { 0x3f3eb880, 0x3639, 0x11ec, \
+			{ 0x9b, 0x9d, 0x0f, 0x3f, 0xc9, 0x46, 0x8f, 0x50 } }
+/*
```

Comment:
For commit "core: pta: APDU pseudo trusted application": add an empty line above